issues/BigtablePerfBenchmarking/BigtablePerfBenchmarking.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.Bigtable.Common.V2" Version="3.0.0" />
```

Comment:
I don't think we should need this reference - it should already be implicit by virtue of having Google.Cloud.Bigtable.V2

Reply:
done